velox/exec/Task.cpp

```diff
@@ -189,7 +190,18 @@ Task::~Task() {
       }
     }
   } catch (const std::exception& e) {
-    LOG(WARNING) << "Caught exception in ~Task(): " << e.what();
+    LOG(WARNING) << "Caught exception in Task " << taskId()
+                 << " destructor: " << e.what();
+  }
+
+  if (!spillDirectory_.empty()) {
```

Comment:
```
Task::cleanupSpillFiles() {
  if (spillDirectory_.empty()) {
    return;
  }
  ...
}
```

Reply:
I don't feel like we need to add yet another method, but that can be done.